src/identity/components/GlobalHeader/GlobalHeader.tsx

```diff
@@ -110,14 +111,21 @@ export const GlobalHeader: FC = () => {
         </FlexBox>
       )}
 
-      {shouldLoadAvatar && (
-        <IdentityUserAvatar
-          email={user.email}
-          firstName={avatarFirstName}
-          lastName={avatarLastName}
-          orgId={org.id}
-        />
-      )}
+      <FlexBox
+        margin={ComponentSize.Large}
+        style={{width: '700px'}}
```

Comment:
What about smaller viewports? On a smaller screen will 700px overwhelm the other elements?

Reply:
offline: tldr: our app doesn't support mobile at the moment so we'll come back to supporting mobile if needed in the future.